Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	
+++ b/src/main/resources/application.properties	
@@ -1,1 +1,2 @@
 spring.datasource.url=jdbc:postgresql://db.qopwjnozklsbcleicxob.supabase.co:5432/postgres?user=postgres&password=M1nt1c2022#.
+spring.jpa.hibernate.ddl-auto=update
Index: src/main/java/co/Mintic/Ciclo3/Ciclo3Application.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/co/Mintic/Ciclo3/Ciclo3Application.java b/src/main/java/co/Mintic/Ciclo3/Ciclo3Application.java
--- a/src/main/java/co/Mintic/Ciclo3/Ciclo3Application.java	
+++ b/src/main/java/co/Mintic/Ciclo3/Ciclo3Application.java	
@@ -1,5 +1,6 @@
 package co.Mintic.Ciclo3;
 
+import co.Mintic.Ciclo3.Entidades.Empresa;
 import net.bytebuddy.build.ToStringPlugin;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
@@ -17,6 +18,13 @@
 		return "Hello world";
 	}
 
+	@GetMapping("/test")
+
+	public String test(){
+		Empresa company = new Empresa("Morfo-Clothing", "Carrera 29 # 78 01", "3507023133","1013605223" );
+		return company.getNombre();
+	}
+
 	public static void main(String[] args) {
 		SpringApplication.run(Ciclo3Application.class, args);
 	}
